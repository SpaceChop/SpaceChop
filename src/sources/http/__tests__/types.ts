import HTTPSourceConfig from '../types';

describe('validate http source', () => {
  it('should require valid source http', () => {
    const config = {};
    expect(() => HTTPSourceConfig.check(config)).toThrowErrorMatchingSnapshot();
  });

  it('should require valid http root url', () => {
    const config = { root: '' };
    expect(() => HTTPSourceConfig.check(config)).toThrowErrorMatchingSnapshot();
  });
});
