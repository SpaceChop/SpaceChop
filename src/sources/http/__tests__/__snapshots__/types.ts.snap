// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`validate http source should require valid http root url 1`] = `"Requires valid url"`;

exports[`validate http source should require valid source http 1`] = `"Expected string, but was undefined"`;
